External/Plugins/FlashDebugger/Controls/DataTreeControl.cs

```diff
@@ -243,120 +247,142 @@ void TreeExpanding(Object sender, TreeViewAdvEventArgs e)
         {
             if (e.Node.Index >= 0)
             {
-                DataNode node = e.Node.Tag as DataNode;
-				if (node.Nodes.Count == 0 && node.Variable != null)
-                {
-					FlashInterface flashInterface = PluginMain.debugManager.FlashInterface;
-					List<DataNode> nodes = new List<DataNode>();
-					List<DataNode> inherited = new List<DataNode>();
-					List<DataNode> statics = new List<DataNode>();
-					int tmpLimit = node.ChildrenShowLimit;
-					foreach (Variable member in node.Variable.getValue().getMembers(flashInterface.Session))
+                ListChildItems(e.Node);
+            }
+        }
+		
+		public void ListChildItems(TreeNodeAdv treeNode) {
+		
+			DataNode node = treeNode.Tag as DataNode;
+			
+			if (node.Nodes.Count == 0 && node.Variable != null)
+			{
+				FlashInterface flashInterface = PluginMain.debugManager.FlashInterface;
+				List<DataNode> nodes = new List<DataNode>();
+				List<DataNode> inherited = new List<DataNode>();
+				List<DataNode> statics = new List<DataNode>();
+				int tmpLimit = node.ChildrenShowLimit;
+				foreach (Variable member in node.Variable.getValue().getMembers(flashInterface.Session))
+				{
+					DataNode memberNode = new DataNode(member);
+					
+					if (member.isAttributeSet(VariableAttribute_.IS_STATIC))
 					{
-						DataNode memberNode = new DataNode(member);
-                        
-						if (member.isAttributeSet(VariableAttribute_.IS_STATIC))
-						{
-							statics.Add(memberNode);
-						}
-						else if (member.getLevel() > 0)
-						{
-							inherited.Add(memberNode);
-						}
-						else
-						{
-							nodes.Add(memberNode);
-						}
+						statics.Add(memberNode);
 					}
-					if (inherited.Count > 0)
+					else if (member.getLevel() > 0)
 					{
-						DataNode inheritedNode = new DataNode("[inherited]");
-						inherited.Sort();
-						foreach (DataNode item in inherited)
-						{
-							inheritedNode.Nodes.Add(item);
-						}
-						node.Nodes.Add(inheritedNode);
+						inherited.Add(memberNode);
 					}
-					if (statics.Count > 0)
+					else
 					{
-						DataNode staticNode = new DataNode("[static]");
-						statics.Sort();
-						foreach (DataNode item in statics)
-						{
-							staticNode.Nodes.Add(item);
-						}
-						node.Nodes.Add(staticNode);
+						nodes.Add(memberNode);
+					}
+				}
+
+				// inherited vars
+				if (inherited.Count > 0)
+				{
+					// list inherited alongside main class members
+					foreach (DataNode item in inherited)
```

Comment:
Can we add an option in FD options to let the dev choose which he wants? I personally find listing inherited side-by-side very useful, since it lets me see properties I care about without going into [inherited] group.